### PR TITLE
Feat :added navbar and hero section

### PR DESCRIPTION
**Pull request by shaikhhamza11**
Screenshot (36)
1)Added header page

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+}
+
+/*header  */
+#header {
+  font-size: 3rem;
+  max-height: 10rem;
+  box-shadow: 0 5px 15px rgba(0, 0, 0, 0.6);
+  color: var(--txt-color-dark);
+  background: var(--bg-primary);
+  /* width: 100%; */
```
Comment (amarphule) [inline css/style.css]: remove comment

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  right: 23px;
+}
+.menu {
+  position: relative;
+}
+
+#menu-list {
+  position: absolute;
+  top: 45px;
+  background: #e9e9e9;
```
Comment (amarphule) [inline css/style.css]: as you are using variables for colors here also for `background` you could use variable

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  flex-direction: column;
+}
+#menu-list ul li {
+  padding: 1rem;
+}
+#navbar .toggle {
+  display: block;
+}
+#account-dropdown + div ul li:first-child {
+  /* background: var(--hover-color); */
```
Comment (amarphule) [inline css/style.css]: remove comment

**Review by amarphule — COMMENTED** [review]
good to go

**Review by Vivek-Py — APPROVED** [review]
Looks good to me!

**Pull request merged**